rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -243,7 +250,7 @@ impl RoadBuilder {
         .take(num_driving_back)
         .collect();
         // TODO Fix upstream. https://wiki.openstreetmap.org/wiki/Key:centre_turn_lane
-        if tags.is("lanes:both_ways", "1") || tags.is("centre_turn_lane", "yes") {
+        for _ in 0..num_driving_both {
```

Comment:
Do we have an example of multiple bidirectional lanes? I'm super curious how that would even work in practice O_o

Reply:
I don't really know. I have seen elsewhere the assumption that there can be at most 1 bothways lane (JOSM Lanes I think). 

The reason for the change is actually that the number of bothway lanes is more specifically what we need to look at here, as unlikely as the viability might be - e.g. it supports `lanes:both_ways=2` just fine, even if it breaks someone else's assumptions.